source/concepts/applications.rst

```diff
@@ -231,52 +176,106 @@ The command does not return anything. This leads to the following signature:
    multiuser/radially-orient-origins(radius = 1.0) -> None
 
 Let a set of players :math:`P = \{P_0, P_1, ... P_{N - 1}\}`, :math:`N` the number of
-players, and :math:`r` the radius given in argument. Then the center's position
-:math:`\mathbf{C}_i` for avatar :math:`i` is computed using polar coordinates converted
-to Cartesian. Each avatar is assigned an angle :math:`\theta_i`:
+players, and :math:`r` the radius given as an argument. Then the center's position
+:math:`\mathbf{C}_p` for avatar :math:`p` is computed using polar coordinates converted
+to Cartesian. Each avatar is assigned an angle :math:`\theta_p`:
 
 .. math::
 
-  \theta_i = \frac{i \times 2 \pi}{N}
+  \theta_p = \frac{ 2 \pi p}{N}
 
-Then the positions is:
+Then each position is:
```

Comment:
Position of what? Is it something like: "Then the position of the center of the server space in each avatar's play space is:"?